children/admin.py

```diff
@@ -45,21 +46,34 @@ class ChildAdmin(admin.ModelAdmin):
         "last_name",
         "birthdate",
         "postal_code",
-        "get_project_year",
+        "get_guardian",
         "created_at",
         "updated_at",
     )
-    list_select_related = ("project",)
-    fields = ("project", "first_name", "last_name", "birthdate", "postal_code")
-    search_fields = ("first_name", "last_name")
+    fields = ("first_name", "last_name", "birthdate", "postal_code")
+    search_fields = (
+        "first_name",
+        "last_name",
+        "birthdate",
+        "guardians__first_name",
+        "guardians__last_name",
+        "guardians__email",
+    )
     inlines = (
         RelationshipInline,
         EnrolmentInline,
         LanguagesSpokenAtHomeInline,
         SubscriptionInline,
     )
+    list_filter = ("project",)
+
+    def get_queryset(self, request):
+        return super().get_queryset(request).prefetch_related("guardians")
 
-    def get_project_year(self, obj):
-        return obj.project.year
+    def get_guardian(self, obj):
+        try:
+            return obj.guardians.all()[0]
```

Comment:
`obj.guardians.first()`? Or is there some reason why we need to have this try-except?

Reply:
`.all()[0]` uses prefetched guardians. Could have added a comment about that.